lib/computer_choice.rb

```diff
@@ -0,0 +1,11 @@
+
+def random(selection, name)
+  comp = ["rock", "paper", "scissors"].sample
+  if (selection == "scissors" && comp == "paper") || (selection == "rock" && comp == "scissors") || (selection == "paper" && comp == "rock")
+    return "#{name} wins"
+  elsif (comp == "scissors" && selection == "paper") || (comp == "rock" && selection == "scissors") || (comp == "paper" && selection == "rock")
```

Comment:
Really clever way of shortening down the if statements for who wins the RPS game! One of your lines does what a paragraph on my file would do, it's the same thing but you're one is shortened down and very pleasing to the eye as opposed to mine which is a bunch of mess.